phpBB/styles/prosilver/template/index_body.html

```diff
@@ -1,79 +1,79 @@
-<!-- INCLUDE overall_header.html -->
+{% INCLUDE 'overall_header.html' %}
 
-<p class="{S_CONTENT_FLOW_END} responsive-center time<!-- IF S_USER_LOGGED_IN --> rightside<!-- ENDIF -->"><!-- IF S_USER_LOGGED_IN -->{LAST_VISIT_DATE}<!-- ELSE -->{CURRENT_TIME}<!-- ENDIF --></p>
-<!-- IF S_USER_LOGGED_IN --><p class="responsive-center time">{CURRENT_TIME}</p><!-- ENDIF -->
+<p class="{{ S_CONTENT_FLOW_END }} responsive-center time{% if S_USER_LOGGED_IN %} rightside{% endif %}">{% if S_USER_LOGGED_IN %}{{ LAST_VISIT_DATE }}{% else %}{{ CURRENT_TIME }}{% endif %}</p>
+{% if S_USER_LOGGED_IN %}<p class="responsive-center time">{{ CURRENT_TIME }}</p>{% endif %}
 
-<!-- EVENT index_body_markforums_before -->
-<!-- IF U_MARK_FORUMS -->
+{% EVENT index_body_markforums_before %}
+{% if U_MARK_FORUMS %}
 	<div class="action-bar compact">
-		<a href="{U_MARK_FORUMS}" class="mark-read rightside" accesskey="m" data-ajax="mark_forums_read">{L_MARK_FORUMS_READ}</a>
+		<a href="{{ U_MARK_FORUMS }}" class="mark-read rightside" accesskey="m" data-ajax="mark_forums_read">{{ lang('MARK_FORUMS_READ') }}</a>
 	</div>
-<!-- ENDIF -->
-<!-- EVENT index_body_markforums_after -->
+{% endif %}
+{% EVENT index_body_markforums_after %}
 
-<!-- INCLUDE forumlist_body.html -->
+{% INCLUDE 'forumlist_body.html' %}
 
-<!-- EVENT index_body_forumlist_body_after -->
+{% EVENT index_body_forumlist_body_after %}
 
-<!-- IF not S_USER_LOGGED_IN and not S_IS_BOT -->
-	<form method="post" action="{S_LOGIN_ACTION}" class="headerspace">
-	<h3><a href="{U_LOGIN_LOGOUT}">{L_LOGIN_LOGOUT}</a><!-- IF S_REGISTER_ENABLED -->&nbsp; &bull; &nbsp;<a href="{U_REGISTER}">{L_REGISTER}</a><!-- ENDIF --></h3>
+{% if not S_USER_LOGGED_IN and not S_IS_BOT %}
+	<form method="post" action="{{ S_LOGIN_ACTION }}" class="headerspace">
+	<h3><a href="{{ U_LOGIN_LOGOUT }}">{{ lang('LOGIN_LOGOUT') }}</a>{% if S_REGISTER_ENABLED %}&nbsp; &bull; &nbsp;<a href="{{ U_REGISTER }}">{{ lang('REGISTER') }}</a>{% endif %}</h3>
```

Comment:
`L_LOGIN_LOGOUT` is a php generated language string.

Reply:
Hi
it can not replace with
{ lang('LOGIN_LOGOUT') }
?

Reply:
No.

`{{ L_LOGIN_LOGOUT }}`

Reply:
To be clear:
https://github.com/phpbb/phpbb/blob/master/phpBB/language/en/common.php

A `LOGIN_LOGOUT` key does NOT exist so the `lang` helper function wouldn't be able to retrieve it. It is a template key built through code, as pointed out by @3D-I.